main.go

```diff
@@ -40,7 +40,8 @@ func main() {
 		FileLimit:         1000,
 		MinFileSize:       sizestr.Bytes(sizestr.MustParse("25MB")),
 		WatchDelay:        3 * time.Second,
-		AccuracyThreshold: 95, //100 is perfect match
+		AccuracyThreshold: 95, //100 is perfect match,
+		Action:            "move",
```

Comment:
`moveAction`

Reply:
Nop. `moveAction` is not defined in `main.go` as not an exported variable.